src/renderer/atlas/AtlasEngine.r.cpp

```diff
@@ -627,34 +623,27 @@ AtlasEngine::CachedGlyphLayout AtlasEngine::_getCachedGlyphLayout(const wchar_t*
         //   --> offsetY = 0
         //   --> scale   = layoutBox.y / (layoutBox.y + left + right)
         //               = 0.69f
-        offset.x = clampedOverhang.left - clampedOverhang.right;
-        offset.y = clampedOverhang.top - clampedOverhang.bottom;
+        offset.x = std::max(0.0f, overhang.left) - std::max(0.0f, overhang.right);

```

Comment:
This changes how overhangs are dealt with; do we need to update the explanatory comment above? Specifically I notice that the "bar diacritic" part has a negative left value...

Reply:
No the big comment is still up to date. This particular change (for `offset.x`) is equivalent to the old code. `clampedOverhang` used to be computed as:
```cpp
const DWRITE_OVERHANG_METRICS clampedOverhang{
    std::max(0.0f, overhang.left),
    std::max(0.0f, overhang.top),
    std::max(0.0f, overhang.right),
    std::max(0.0f, overhang.bottom),
};
```